pennylane/devices/default_qubit_legacy.py

```diff
@@ -78,13 +79,13 @@ def _get_slice(index, axis, num_axes):
 
 # pylint: disable=unused-argument
 class DefaultQubitLegacy(QubitDevice):
-    """Default qubit device for PennyLane.
+    r"""Default qubit device for PennyLane.
 
     .. warning::
 
-        This is the legacy implementation of DefaultQubit. It has been replaced by
-        ``qml.devices.DefaultQubit``, which can be accessed with the familiar constructor,
-        ``qml.device("default.qubit")``.
+        This is the legacy implementation of DefaultQubit and is deprecated. It has been replaced by
+        :class:`~.devices.DefaultQubit`, which can be accessed with the familiar constructor,
+        ``qml.device("default.qubit")``, and now supports backpropagation.
```

Comment:
Why is backprop support in `DefaultQubit` highlighted so consistently? It's been around forever so it seems pointless.